Src/java/quick/src/main/resources/org/hl7/fhir/fhir-modelinfo-4.0.0.xml

```diff
@@ -1354,7 +1354,7 @@
     <ns4:typeInfo xsi:type="ns4:ClassInfo" name="AssertionResponseTypes" retrievable="false" baseType="FHIR.Element">
         <ns4:element name="value" elementType="System.String"/>
     </ns4:typeInfo>
-    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="OperationOutcome" retrievable="true" baseType="FHIR.DomainResource">
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="OperationOutcome" retrievable="true" baseType="FHIR.DomainResource" primaryCodePath="issue.code">
```

Comment:
`issue.code` is technically a `FHIR.IssueType`, which is technically a `String`.  I think that's OK, but just mentioning it in case. (FHIR 4.0.0 `OperationOutcome`)